types.go

```diff
@@ -478,3 +478,16 @@ type FlashbotsPrivateTxPreferences struct {
 type FlashbotsCancelPrivateTransactionRequest struct {
 	TxHash string `json:"txHash"`
 }
+
+type FlashbotsSendBundleCrossRollupRequest struct {
+	Txs               map[string]string `json:"txs"`
```

Comment:
You might want to consider []byte for the value type here. I had some problems before when marshaling to and from json due to certain values getting mapped to escape characters which messes up processing.

Reply:
We use hexutil.Encode to encode the bytes not directly from go string([]bytes), go ethereum does that the same way so don't worry